pkg/apis/db/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -374,6 +395,29 @@ func (in RackSlice) DeepCopy() RackSlice {
 	return *out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ServicePolicy) DeepCopyInto(out *ServicePolicy) {
+	*out = *in
+	if in.Annotations != nil {
+		in, out := &in.Annotations, &out.Annotations
+		*out = make(map[string]string, len(*in))
+		for key, val := range *in {
+			(*out)[key] = val
+		}
+	}
+	return
+}
```

Comment:
We should not have the same code for both pod and service policies. Let’s refactor it